pysatModels/tests/test_utils_extract.py

```diff
@@ -362,7 +362,7 @@ def test_success_for_some_already_ran_data(self):
         return
 
 
-@pytest.mark.skipif(pysat_version_major <= 3 & (pysat_version_minor < 1),
+@pytest.mark.skipif(pysat_version_major <= 3 and (pysat_version_minor < 1),
```

Comment:
In theory, wouldn't pysat 2.2 be run with this logic?

Reply:
pytest skips the test if the check evaluates True.

I wrote the code the wrong way the first time 🙈

Reply:
True and False = False for pysat 2.2

Reply:
I guess technically since pysat 3.0 is the minimum req in develop, this case will never occur on the CI side.